### PR TITLE
Wdi10 updates

### PR DESCRIPTION
**Pull request by amaseda**

### CODE REVIEW

File: flexbox.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
...
+
+## How It Works (10 minutes / 0:10)
+
+![flexbox diagram](img/flexbox-diagram.jpg)
+
+When you declare `display: flex` on a container, it becomes a **flex container**.
+
+First, you use `flex-direction` to indicate whether you want the items in the container -- the **flex items** -- to "read" left-to-right (`row`), right-to-left (`row-reverse`), top-to-bottom (`column`), **or** bottom-to-top (`column-reverse`).
+
+When you specify a flex-direction, you can think of it as placing an axis in that direction across your flex container. So if you use `flex-direction: row` or `row-reverse`, this **main axis** will be the same as the X-axis (horizontal) on a graph. Otherwise, it'll be the Y-axis.
```
Comment (jshawl) [inline flexbox.md:114]: diagram might be nice here

File: sass.md [hunk 2]
```diff
@@ -37,332 +37,86 @@ to generate the css (by clicking "View Compiled").
 - [Forest](http://codepen.io/jshawl/pen/cJjIm)
 - [Space Invader](http://codepen.io/jshawl/pen/cnyrJ)
```
Comment (jshawl) [inline sass.md:38]: please find pens that are not just me :)

File: flexbox.md (hunk 1 above)
Comment (amaseda) [inline flexbox.md:114]: @jshawl I have one on line 104, not sure which one's better though. Thoughts?

**Pull request merged**